### DISCUSSION TITLE
モック作成までのタスク整理

### DISCUSSION THREAD

**Discussion by tecsoc** [tech-creative-club/disaster-posts#32] [General]
一旦この辺かな？
- state管理
- UI調整
- デプロイ（GitHub Actions？）

**Comment by Kokohachi** [comment]
- state管理
モックのときはとりあえずuseStateでいい気がします。
- UI調整
本当はfigmaとかで作りたいが、私はよくそのまんま書いちゃいます。（個人的好みは角丸）
- デプロイ
Next.jsなんでとりまVercelで良い気がします。
